{{cookiecutter.github_repo_name}}/tools/restore-filestore.sh

```diff
@@ -0,0 +1,13 @@
+# Prepare arguments
+while getopts d:f:r: flag
+do
+    case "${flag}" in
+        d) dbname=${OPTARG};;
+        f) filestorepath=${OPTARG};;
+        r) dockerrun=${OPTARG};;
+    esac
+done
+echo "Restoring filestore $filestorepath for database $dbname in docker $dockerrun"
+# Move files
+echo "Moving files to docker image"
+docker cp $filestorepath/. $dockerrun:/var/odoo/filestore/$dbname/
```

Comment:
Puoi verificare che non sia necessario correggere i permessi dei file quando si copiano all'interno del filestore? Francamente non so quale utente usa docker nel ripristino.

Inoltre, puoi verificare che docker cp sia in grado di gestire in modo intelligente il caso in cui il filestore viene ripristinato su un'istanza con filestore già esistente? Idealmente dovrebbe mergiare i due contenuti alla rsync.

Reply:
Entrambe le prove le ho fatte perché lo script l'ho testato su un db restorato giorni fa ma del quale non avevo portato dentro il filestore.